docusaurus.config.js

```diff
@@ -129,6 +129,20 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
+        appId: "OQHBL9CWH9",
+        apiKey: "042138591ea150790b1f40a365755caf",
```

Comment:
do you know where we deploy this? @Nadai2010 , seems like we might need to extract this to an env. if we do need ENVs we can inform @mimisavage here.

sorry @mimisavage for the delay, normally its not very good practice to hardcode api keys in a repo haha, people can abuse it, let's not rush things

Reply:
did you have to generate some keys here ? @mimisavage

Reply:
> did you have to generate some keys here ? @mimisavage

yes i use algolia.
the this the crawler 

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/cf30f2ea-7fae-4d7f-8dd5-1694af30d304" />

Reply:
I will add to env and also share the generated api privately via telegram